tests/test_core.py

```diff
@@ -6954,6 +6954,7 @@ def test_embind_unsigned(self):
 
   def test_embind_val(self):
     self.emcc_args += ['--bind']
+    self.set_setting('ABORT_ON_WASM_EXCEPTIONS', 0) # The test handles exceptions itself
```

Comment:
Is this still needed given the default is 0?

Reply:
Nope, removing.